hello_world.cpp

```diff
@@ -0,0 +1,7 @@
+// Note: this file is just to mark this project as C++ related.
+#include <iostream>
+
+int main()
+{
+    std::cout << "Hello, world!\n";
```

Comment:
@nabijaczleweli: You left a number of unnecessarily rude comments in some of the issues here.

I'd like to note that it's okay for people to be wrong and the way to handle this is not to laugh at them and tell them how stupid they are, but to leave out personal insults and keep just the discussion about the code itself. Nobody needs to feel bad or get told that they can't read.

Nitpicky one-liners aren't all that useful to begin with, but especially not when they're scaring people away more than they fix real issues. Yes, `std::endl` is overrated. No, this is not a performance-critical hot path but a file only intended to tag a project with a particular language. None of this code is even remotely important enough to put people down over it. If you need to have an outlet for your temper, keep it to your own issue queues.